VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: 


BO_ 668 Wheel_Speed_Rear: 8 Vector__XXX
 SG_ Wheel_Speed_RR : 23|16@0+ (0.005,0) [0|0] "km/h" Vector__XXX
 SG_ Wheel_Speed_RL : 7|16@0+ (0.005,0) [0|0] "km/h" Vector__XXX

BO_ 666 Wheel_Speed_Front: 7 Vector__XXX
 SG_ Wheel_Speed_FL : 23|16@0+ (0.005,0) [0|0] "km/h" Vector__XXX
 SG_ Wheel_Speed_FR : 7|16@0+ (0.005,0) [0|0] "km/h" Vector__XXX





BA_DEF_ BO_  "GenMsgCycleTime" INT 0 65535;
BA_DEF_DEF_  "GenMsgCycleTime" 0;





